core.go

```diff
@@ -159,17 +159,22 @@ func (c *Core) init(name, dir string, relationships []string) (err error) {
 	return
 }
 
+func (c *Core) newReflectValue() (value reflect.Value) {
```

Comment:
Is this called from every transaction, or just on init?

Reply:
This is called every time we want to initialize a new entry to append to the slice.

Reply:
Is reflection going to cause slowdowns for large slices?

Reply:
@dhalman It's just pulled into this function, we have been already doing it before (look at line 164 in old file)
P.S. I do think it causes slowdowns, however.

Reply:
Yeah now we are using newReflectValue -> newEntryValue -> newValueFromBytes. It's a refactor, took a minute to figure out...